python/arcticdb/adapters/azure_library_adapter.py

```diff
@@ -55,6 +64,7 @@ def __init__(self, uri: str, encoding_version: EncodingVersion, *args, **kwargs)
         )
         self._container = self._query_params.Container
         self._ca_cert_path = self._query_params.CA_cert_path
+        self._ca_cert_dir = self._query_params.CA_cert_dir
```

Comment:
We don't persist this right? The "masking override" will handle it without further code changes?

Reply:
https://github.com/man-group/ArcticDB/blob/1eb81673e1a91ae227a4e59ccdfef5e83e8cf08e/python/arcticdb/adapters/azure_library_adapter.py#L129C31-L129C31
https://github.com/man-group/ArcticDB/blob/1eb81673e1a91ae227a4e59ccdfef5e83e8cf08e/cpp/arcticdb/storage/storage_override.hpp#L132C1-L132C1
It has been addressed